nginx/server.conf

```diff
@@ -0,0 +1,42 @@
+server {
+  listen 80;
+  listen [::]:80;
+
+  server_name ${NGINX_HOST};
+
+  return 301 https://${NGINX_HOST}$request_uri;
+}
+
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+
+  server_name ${NGINX_HOST};
+
+  ssl_certificate     /certs/fullchain.pem;
+  ssl_certificate_key /certs/privkey.pem;
+  ssl_dhparam         /certs/dhparams.pem;
+  ssl_prefer_server_ciphers on;
+  ssl_ciphers EECDH+CHACHA20:EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
+  ssl_protocols TLSv1.3;
```

Comment:
This can make problems e.g. on ipads - we might want to use the intermediate config from here

https://ssl-config.mozilla.org/

Reply:
wow. didn't know mozilla offered this. Fixed in e2057698a839588b2c60c0e9a267c7c8f9945c02